modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -111,6 +112,28 @@ def diff(rep_data)
         end
       end
 
+      def self.get_suffix(first_name, last_name)
+        suffixes = %w[H.M H.I.H H.R.H Dr Esq Jr Sr III II I AB BA BS BE BFA BTech LLB BSc MA MS MFA LLM
+                      MLA MBA MSc MEng JD MD DO PharmD DMin PhD EdD DPhil DBA LLD EngD]
+        suffixes.detect { |suff| last_name.include?(suff) || first_name.include?(suff) }
```

Comment:
As written, we will also include prefixes, e.g.,

```rb
"Dr. Phil".include?('Dr')
=> true
```

Is that intended/desired?

Reply:
Should we also make the check case insensitive as we do for first and last name?